constants.py

```diff
@@ -1,7 +0,0 @@
-import os
```

Comment:
:O didn't mean to remove that, but I guess it's fine, since os is either way imported in scheduler.py which inturn calls this, control flow works it out.. :P